test/video.test.js

```diff
@@ -11,8 +11,9 @@ describe('Video', () => {
 	beforeEach(() => {
 		containerEl = document.createElement('div');
 		containerEl.setAttribute('data-o-component', 'o-video');
-		containerEl.setAttribute('data-o-video-id', '4084879507001');
```

Comment:
Yep, although there are some assertions based around this exact uuid being passed in which would fail.

I've added another couple of tests to demonstrate how this is backwards compatible: https://github.com/benbarnett/o-video/commit/f5857de0b7d2db09b61ae9bcfc922158b89b7255

Do you think that helps clarity?